File: calculators/calc.py
#!/usr/bin/env python
import sys
import operator

operator_dict = {
    "+": operator.add,
    "-": operator.sub,
    "*": operator.mul,
    "/": operator.div,
}

def main():
    left, op, right = sys.argv[1].split()
    answer = operator_dict[op](int(left), int(right))

    print answer
    

if __name__ == "__main__":
    main()

File: calculators/calc.rb
#!/usr/bin/env ruby
left, op, right = ARGV[0].split

allowed = ["+","-","*","/"]

if allowed.include? op
  answer = left.to_f.send(op, right.to_f)
  puts answer
else
  exit 1
  puts "Error"
end
